### PR TITLE
update spec for DepthToSpace operation

### PR DESCRIPTION
**Pull request by pelszkow**
### Details:
 - *update spec for DepthToSpace operation*

### Tickets:
 - *45786*

### CODE REVIEW

File: docs/ops/movement/DepthToSpace_1.md [hunk 1]
```diff
@@ -6,6 +6,26 @@
 
 **Short description**: *DepthToSpace* operation rearranges data from the depth dimension of the input tensor into spatial dimensions of the output tensor.
 
+**Detailed description**
+
+*DepthToSpace* operation permutes elements from the input tensor with shape `[N, C, D1, D2, ..., DK]`, to the output tensor where values from the input depth dimension (features) `C` are moved to spatial blocks in `D1`, ..., `DK`. Refer to the [ONNX* specification](https://github.com/onnx/onnx/blob/master/docs/Operators.md#DepthToSpace) for an example of the 4D input tensor case.
+
```
Comment (jdanieck) [inline docs/ops/movement/DepthToSpace_1.md:12]: We generally avoid referring other frameworks, hence I'd remove this sentence with ONNX spec.

**Review by jdanieck — APPROVED** [review]

**Comment by ggalieroc** [comment]
Fine tune: to be consistent with attributes description, either add the attribute name in both or remove it. Right now it looks strange.

**Attributes**

* *block_size*
  * **Description**: specifies the size of the value block to be moved. The depth dimension size must be evenly divided by `block_size ^ (len(input.shape) - 2)`.

* *mode*
  * **Description**: specifies how the input depth dimension is split to block coordinates and the new depth dimension.

File: docs/ops/movement/DepthToSpace_1.md [hunk 2]
```diff
@@ -28,31 +48,15 @@
...
 **Outputs**
 
-*   **1**: permuted tensor with shape `[N, C / block_size ^ K, D1 * block_size, D2 * block_size, ..., DK * block_size]`.
-
-**Detailed description**
-
-*DepthToSpace* operation permutes elements from the input tensor with shape `[N, C, D1, D2, ..., DK]`, to the output tensor where values from the input depth dimension (features) `C` are moved to spatial blocks in `D1`, ..., `DK`. Refer to the [ONNX* specification](https://github.com/onnx/onnx/blob/master/docs/Operators.md#DepthToSpace) for an example of the 4D input tensor case.
-
-The operation is equivalent to the following transformation of the input tensor `data` with `K` spatial dimensions of shape `[N, C, D1, D2, ..., DK]` to *Y* output tensor. If `mode = blocks_first`:
+*   **1**: permuted tensor with type T and shape `[N, C / block_size ^ K, D1 * block_size, D2 * block_size, ..., DK * block_size]`.
```
[the remark this answers is not in the document]
Comment (pelszkow) [reply docs/ops/movement/DepthToSpace_1.md:55]: done

File: docs/ops/movement/DepthToSpace_1.md (hunk 1 above)
Comment (pelszkow) [reply docs/ops/movement/DepthToSpace_1.md:12]: done

**Review by jdanieck — APPROVED** [review]

**Review by ggalieroc — APPROVED** [review]

**Comment by pelszkow** [comment]
> * specifies the size of the value block to be moved. The depth dimension size must be evenly divided by

Done

**Review by ggalieroc — APPROVED** [review]

**Comment by pelszkow** [comment]
@lazarevevgeny, can you look on this PR and merge it?

**Review by lazarevevgeny — APPROVED** [review]

**Pull request merged**